src/fprime/util/cli.py

```diff
@@ -179,4 +225,54 @@ def add_special_parsers(
         action="store_true",
         help="Tells the new command to generate a port",
     )
-    return {"hash-to-file": hash_to_file, "info": print_info, "new": template}
+
+    # Code formatting with clang-format
+    format_parser = subparsers.add_parser(
+        "format",
+        help=help_text.short("format"),
+        description=help_text.long("format"),
+        parents=[common],
+        add_help=True,
```

Comment:
Typically we don't set the `add_help` flag to `True`.  This is done at the top level.

Reply:
Changed back to False